examples/eks/main.tf

```diff
@@ -66,7 +66,7 @@ resource "aws_cognito_user_in_group" "add_admin_argocd_admin" {
 */
 
 module "eks" {
-  source = "git::https://github.com/camptocamp/devops-stack.git//modules/eks/aws?ref=v1"
+  source = "git::https://github.com/camptocamp/devops-stack.git//modules/eks/aws?ref=v2.0.1"
```

Comment:
This line is no longer true, we need to change to the new cluster module repository.